hypha/apply/users/forms.py

```diff
@@ -51,6 +51,11 @@ def __init__(self, *args, **kwargs):
 
 
 class CustomUserAdminFormBase:
+    error_messages = {
+        "duplicate_username": _("A user with that email already exists."),
+        "password_mismatch": _("The two password fields didn't match."),
+    }
+
```

Comment:
Moved it here so it becomes active on the edit form as well.

Reply:
👍🏼